### ISSUE TITLE
Lazy aggregated batch verification

### ISSUE THREAD

**Issue by dapplion** [ChainSafe/lodestar#3612]
**Is your feature request related to a problem? Please describe.**

Reduce load for busy gossip channels

**Describe the solution you'd like**

See https://github.com/status-im/nimbus-eth2/pull/3212

**Describe alternatives you've considered**

N/A

**Comment by philknows** [comment]
As stated in the Mar 18 planning call, we should target gossipsub upgrade first, then implement this.

**Comment by wemeetagain** [comment]
@twoeths I think this is our same-message optimisation? In that case this can be closed

**Comment by twoeths** [comment]
yes but we only implemented for attestation, need to also implement for sync committee messages
